### PR TITLE
Add description to commander CLI help

### PR DESCRIPTION
**Pull request by cvrebert**

### CODE REVIEW

File: bin/jscs [hunk 1]
```diff
@@ -13,6 +13,7 @@ var program = require('commander');
 program
     .version(require('../package.json').version)
     .usage('[options] <file ...>')
+    .description('Programmatically enforce JavaScript code style guides')
```
Comment (mikesherov) [inline bin/jscs]: A code style linter for programmatically enforcing your style guide.

Comment (cvrebert) [inline bin/jscs]: Can you just make that tweak as part of your rebase when you merge this?

**Comment by cvrebert** [comment]
Fine, changed & rebased.

**Comment by mikesherov** [comment]
Thanks again!

**Pull request closed without merging**